server/api/products.js

```diff
@@ -0,0 +1,62 @@
+const Sequelize = require('sequelize');
+const router = require('express').Router()
+const { Product } = require('../../db')
+
+router.get('/', async function (req, res, next){
+	try {
+		const products = await Product.findAll()
+  		res.status(200).send(products)
+	} catch (error) {
+		console.log(error)
+		next(error)
+	}
+});
+
+router.get('/:productId', async function (req, res, next){
+	try {
+		const product = await Product.findOne({
+			where: {
+			  id: req.params.productId
+			},
+		})
+  		res.status(200).send(product)
+	} catch (error) {
+		console.log(error)
+		next(error)
+	}  	
+});
+
+router.post('/', function (req, res, next) {
+	Product.create(req.body)
```

Comment:
It was mentioned previously in class that just apply req.body into a create method is a security vulnerability. 
You should list out the properties we're creating with and take their values individually from the req.body

Reply:
Incorporated. 
router.post('/', function (req, res, next) {
	const { name, description, inventory, price } = req.body;
	if (!name) {
        return res.status(400).send('Invalid request; name required');
    }
	Product.create({name, description, inventory, price})
	.then(product => res.status(201).send(product))
	.catch(next);
  });